src/main/java/org/cactoos/text/CapitalizedText.java

```diff
@@ -53,15 +53,15 @@ public CapitalizedText(final Text text) {
     @Override
     public String asString() throws IOException {
         return new JoinedText(
-            new TransformedIterable<String, Text>(
+            new TransformedIterable<Text, Text>(
                 new SplitText(this.origin),
-                new Func<String, Text>() {
+                new Func<Text, Text>() {
                     @Override
-                    public Text apply(final String input) throws Exception {
+                    public Text apply(final Text input) throws Exception {
```

Comment:
Maybe i'm a bit picky here, but it took me about 30 seconds to really understand what this func does. Maybe we can externalize this into their own private method named capitalizeWord() or use a variable for the function, that is named something like that.